Source/Blazorise/Components/MessageAlert/MessageAlert.razor.cs

```diff
@@ -271,6 +272,16 @@ protected virtual ModalSize Size
     /// </summary>
     [Inject] protected IMessageService MessageService { get; set; }
 
+    /// <summary>
+    /// Gets or sets the DI registered <see cref="ITextLocalizerService"/>.
+    /// </summary>
+    [Inject] protected ITextLocalizerService LocalizerService { get; set; }
+
+    /// <summary>
+    /// Gets or sets the DI registered <see cref="ITextLocalizer{MessageProvider}"/>.
+    /// </summary>
+    [Inject] protected ITextLocalizer<MessageProvider> Localizer { get; set; }
```

Comment:
I imagine MessageProvider inherits this, right?

Reply:
I mean the MessageAlert implementation.

Reply:
MessageAlert will be removed eventually so it's better to just name it MessageProvider.